octopusdeploy/resource_project.go

```diff
@@ -93,6 +93,8 @@ func resourceProject() *schema.Resource {
 			"deployment_step_windows_service": getDeploymentStepWindowsServiceSchema(),
 			"deployment_step_iis_website":     getDeploymentStepIISWebsiteSchema(),
 			"deployment_step_inline_script":   getDeploymentStepInlineScriptSchema(),
+			"deployment_step_kubernetes_helm": getDeploymentStepKubernetesHelmSchema(),
```

Comment:
@MJRichardson or @zentron  You know this best, what do you think about this naming? Should it be 'deployment_step_kubernetes_upgrade_helm_chart` instead? I'm guessing we may have have `_delete_helm_chart` or something in the future.

The YAML one seems ok.

Reply:
a delete helm could be possible 🤔 , though its so simple, id probably just lean towards encouraging the use of a kubectl script step unless there was a need/confusion.

If we wanted to make it fully qualified how about instead `deployment_step_kubernetes_helm_upgrade` so if we do have another action its just end end that changes.
It probably cant hurt to use that full name anyway even if its the only one just to make it clear what operation that step is explicitly doing.